ena_upload/templates/ENA_template_vir_sample.xml

```diff
@@ -1,4 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<?python
+import pandas as pd
+def attributetest(row, column):
+    emptyvalue = [' ', 'nan', 'na']
```

Comment:
what about `''`

Reply:
empty cells become an NaN object with pandas, which is detected with the pd.notna() function

Reply:
ok, I'm just worried what happend when a cell contains `'    '`

Reply:
When it only contains a space than that cell will be ignored and not be put in the template. Should this be different better?

Reply:
Sorry that I already bumped a version, we wanted to make some improvements for Estonia as fast as possible and did tests with all the optional fields

Reply:
@bedroesb don't be sorry. Really this PR was great. I just wanted to add some comments, maybe for the next version.

Fast as possible is always good! Rock on!

Reply:
> When it only contains a space than that cell will be ignored and not be put in the template. Should this be different better?

sorry github does not render it correctly, is that also working if there are 5 spaces?

Reply:
ow good catch ... no it doesn't, I found this python function `str.isspace()` that could check for it. I will add this with another version. I would like to support more sample types too in the future.